master/getting-started/docker/installation/manual.md

```diff
@@ -18,46 +18,47 @@ title: Installing Calico for Docker
    sudo chmod +x /usr/local/bin/calicoctl
    ```
 
-2. Configure access to your etcd cluster, [calicoctl - etcd datastore](/{{page.version}}/reference/calicoctl/setup/etcdv3).
-3. Launch `calico/node`:
+1. Configure access to your etcd cluster, [calicoctl - etcd datastore](/{{page.version}}/reference/calicoctl/setup/etcdv3).
+
+1. Launch `{{site.nodecontainer}}`:
 
    ```
    sudo calicoctl node run --node-image={{site.imageNames["node"]}}:{{site.data.versions[page.version].first.title}}
    ```
 
-Check that `calico/node` is now running:
+1. Check that `{{site.nodecontainer}}` is now running:
```

Comment:
I think this should be `{{site.noderunning}}`.

Reply:
Yes, I think you are probably right.